package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "thanhnguyen-html-css-practice-one",
+  "version": "1.0.0",
+  "description": "This is Thanh Nguyen repositories of html/css practice one",
+  "main": "commitlint.config.js",
+  "devDependencies": {
+    "parcel": "^2.9.3",
+    "parcel-bundler": "^1.12.5"
+  },
+  "scripts": {
+    "start": "parcel src/index.html"
+  },
+  "author": "Thanh Nguyen",
+  "license": "ISC",
+  "dependencies": {
+    "bases": "^0.2.1",
+    "components": "^0.1.0",
+    "layouts": "^3.0.2",
+    "utils": "^0.3.1"
```

Comment:
@mthanh2209 Why you need all of these things in dependencies?

Reply:
Sorry, I don't know. When I build Parcel it looks like this

Reply:
sorry, i don't know. When i build Parcel it looks like this.